composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/chat/CreateGroupChatScreen.kt

```diff
@@ -0,0 +1,160 @@
+package com.github.familyvault.ui.screens.main.chat
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Switch
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import cafe.adriel.voyager.core.screen.Screen
+import cafe.adriel.voyager.navigator.LocalNavigator
+import cafe.adriel.voyager.navigator.currentOrThrow
+import com.github.familyvault.forms.GroupChatCreationForm
+import com.github.familyvault.models.FamilyMember
+import com.github.familyvault.models.enums.FormSubmitState
+import com.github.familyvault.services.IChatService
+import com.github.familyvault.services.IFamilyGroupService
+import com.github.familyvault.ui.components.UserAvatar
+import com.github.familyvault.ui.components.ValidationErrorMessage
+import com.github.familyvault.ui.components.overrides.Button
+import com.github.familyvault.ui.components.overrides.TextField
+import com.github.familyvault.ui.components.overrides.TopAppBar
+import com.github.familyvault.ui.components.typography.Headline3
+import com.github.familyvault.ui.components.typography.Paragraph
+import com.github.familyvault.ui.theme.AdditionalTheme
+import familyvault.composeapp.generated.resources.Res
+import familyvault.composeapp.generated.resources.chat_create_group_button_content
+import familyvault.composeapp.generated.resources.chat_create_new
+import familyvault.composeapp.generated.resources.chat_set_group_name
+import familyvault.composeapp.generated.resources.chat_create_group_members
+import kotlinx.coroutines.launch
+import org.jetbrains.compose.resources.stringResource
+import org.koin.compose.koinInject
+
+class CreateGroupChatScreen : Screen {
```

Comment:
Tutaj podobnie co mówiłem w formularzu, lepiej nazwać ten screen `CreateGroupEditScreen`.

Aby rozróżnić operacje dodawania i usuwania na danym screenie można np. przyjmować model `ChatThread` i jeżeli jest nullem to wiadomo wtedy że jest to operacja tworzenia